Wrangler2 release (#3909)
cloudflare/cloudflare-docs: content/workers/wrangler/getting-started.md

---
pcx-content-type: how-to
title: Getting started
weight: 1
---

> ## Getting started with 🤠 Wrangler

Reviewer: ```suggestion
## Getting started with Wrangler
```

`wrangler` is a command line tool for building [Cloudflare Workers](https://workers.cloudflare.com/).

## Quick Start

```sh

> # Make a javascript file
> $ echo "export default { fetch() { return new Response('hello world') } }" > index.js
> # try it out
> $ npx wrangler dev index.js
> # and then publish it
> $ npx wrangler publish index.js --name my-worker
> # visit https://my-worker.<your workers subdomain>.workers.dev

Reviewer 2: ```suggestion
# Generate a new project
npx wrangler init my-worker
# try it out
cd my-worker && npx wrangler dev
# and then publish it
npx wrangler publish
# visit https://my-worker.<your workers subdomain>.workers.dev
```

```

## Installation:

```bash

> $ npm install wrangler

Reviewer 3: `npm install -g wrangler`?

> $ npm install -g wrangler

Reviewer 2: ```suggestion
$ npm install wrangler --save-dev
```
